.agents/structure.md

```
@@ -60,14 +59,13 @@ packages/[name]/
 | `query` | `@routup/query` | Query string parsing | `qs` |
 | `rate-limit` | `@routup/rate-limit` | In-memory rate limiter | — |
 | `rate-limit-redis` | `@routup/rate-limit-redis` | Redis rate-limit adapter | `@routup/rate-limit`, `ioredis` |
-| `swagger` | `@routup/swagger` | Swagger UI serving | `@routup/assets`, `@routup/swagger-preset` |
-| `swagger-preset` | `@routup/swagger-preset` | Swagger HTML/asset preset | `swagger-ui-dist` |
+| `swagger` | `@routup/swagger` | Swagger UI + OpenAPI generator (bundles the `@trapi/metadata` preset) | `@routup/assets`, `@trapi/metadata`, `@trapi/swagger`, `swagger-ui-dist` |
 
 ## Dependency Layers
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)